app.js

```diff
@@ -19,13 +21,19 @@ app.get("/api/articles", getArticles);
 
 app.get("/api/articles/:article_id/comments", getCommentByArticleId);
 
+app.post("/api/articles/:article_id/comments", postCommentByArticleId);
+
 app.all("*", getError);
 
 app.use((err, req, res, next) => {
   if (err.status && err.msg) {
     res.status(err.status).send({ msg: err.msg });
   } else if (err.code === "22P02") {
     res.status(400).send({ msg: "Bad Request" });
+  } else if (err.code === "23502") {
+    res.status(400).send({ msg: "Post Missing Data" });
+  } else if (err.code === "23503") {
+    res.status(404).send({ msg: "Not Found" });
   } else {
     res.status(500).send({ msg: "Internal Server Error" });
```

Comment:
I would suggest making these into functions in a separate folder that you can reference at this point, because they're taking up a lot of space in this file :)